docs/sample_inventories/rbac_sso_c3.yml

```diff
@@ -0,0 +1,108 @@
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+
+    ## TLS Configuration - Custom Certificates
+    ssl_enabled: true
+    #### SASL Authentication Configuration - Choose the one that suit your requirements and configure according ####
+    ## By default there will be no SASL Authentication
+    ## For SASL/PLAIN uncomment this line:
+    sasl_protocol: plain
+    ## For SASL/SCRAM uncomment this line:
+    # sasl_protocol: scram
+    ## For SASL/GSSAPI uncomment this line and see Kerberos Configuration properties below
+    # sasl_protocol: kerberos
+
+    ## RBAC Configuration
+    rbac_enabled: true
+
+    ## LDAP CONFIGURATION
+    kafka_broker_custom_properties:
+      ldap.java.naming.factory.initial: com.sun.jndi.ldap.LdapCtxFactory
+      ldap.com.sun.jndi.ldap.read.timeout: 3000
+      ldap.java.naming.provider.url: ldaps://ldap1:636
+      ldap.java.naming.security.principal: uid=mds,OU=rbac,DC=example,DC=com
+      ldap.java.naming.security.credentials: password
+      ldap.java.naming.security.authentication: simple
+      ldap.user.search.base: OU=rbac,DC=example,DC=com
+      ldap.group.search.base: OU=rbac,DC=example,DC=com
+      ldap.user.name.attribute: uid
+      ldap.user.memberof.attribute.pattern: CN=(.*),OU=rbac,DC=example,DC=com
+      ldap.group.name.attribute: cn
+      ldap.group.member.attribute.pattern: CN=(.*),OU=rbac,DC=example,DC=com
+      ldap.user.object.class: account
+
+    ## LDAP USERS
+    mds_super_user: mds
+    mds_super_user_password: password
+    kafka_broker_ldap_user: kafka_broker
+    kafka_broker_ldap_password: password
+    schema_registry_ldap_user: schema_registry
+    schema_registry_ldap_password: password
+    kafka_connect_ldap_user: connect_worker
+    kafka_connect_ldap_password: password
+    ksql_ldap_user: ksql
+    ksql_ldap_password: password
+    kafka_rest_ldap_user: rest_proxy
+    kafka_rest_ldap_password: password
+    control_center_ldap_user: control_center
+    control_center_ldap_password: password
+
+    ## (OPTIONAL) SYSTEM ADMINS (SYSTEM ADMIN ROLEBINDINGS)
+    # A list of principals can be specified for all the componentes using 'rbac_component_additional_system_admins:'
+    # Or individual components [component]_additional_system_admins (i.e. schema_registry_additional_system_admins)
+
+    ## Varibles to enable SSO in Control Center
+    sso_mode: oidc
+    # necessary configs in MDS server for sso in C3
+    sso_groups_claim: groups
+    sso_sub_claim: sub
+    sso_groups_scope: groups # scope is optional, depending on the Idp
+    sso_issuer_url: <issuer url>
+    sso_jwks_uri: <jwks uri>
+    sso_authorize_uri: <OAuth authorization endpoint>
+    sso_token_uri: <IdP token endpoint>
+    sso_client_id: <client id>
+    sso_client_password: <client password>
```

Comment:
Could you also add a corresponding setting for `confluent.oidc.idp.refresh.token.enabled`

Reply:
added in the sample inventory